repos/fdbt-site/fdbt-types/matchingJsonTypes.ts

```diff
@@ -226,6 +226,7 @@ export interface ReturnTicket extends BasePointToPointTicket {
     inboundFareZones: FareZone[];
     outboundFareZones: FareZone[];
     returnPeriodValidity?: ReturnPeriodValidity;
+    additionalServices?: SelectedService[];
```

Comment:
In the types repo we have this as AdditionalService[] - seems like there's a mismatch.

Reply:
used SelectedService for consistency